script/build-translations.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
```

Comment:
Nice! [It looks like gulp has a module for deep-merging json files.](https://www.npmjs.com/package/gulp-merge-json) I could port this as an initial step.